docs/user/firststeps.rst

```diff
@@ -8,7 +8,8 @@ First Steps
     :keywords: hello world, first steps, starter, primer
 
 To try out Renga, you first need a platform to connect to: see :ref:`setup`
-for instructions on how to get one running on your own machine in a few minutes.
+for instructions on how to get one running on your own machine in a few minutes. 
+You will also need Python with version 3.x.x.
```

Comment:
maybe add it as a note? it will stand out more, e.g. 

```
.. note::
    You will also need Python with version 3.x.x.
```

Reply:
Okay, can do